src/components/Home/TransactionCard/TransactionDetailModal/LNDetails/LNDetails.tsx

```diff
@@ -1,39 +1,45 @@
-import { FC } from "react";
+import { FC, useContext } from "react";
 import { useTranslation } from "react-i18next";
 import { ReactComponent as ClipboardIcon } from "../../../../../assets/clipboard-copy.svg";
+import useClipboard from "../../../../../hooks/use-clipboard";
+import { Transaction } from "../../../../../models/transaction.model";
+import { AppContext } from "../../../../../store/app-context";
+import {
+  convertMSatToBtc,
+  convertMSatToSat,
+  convertToString,
+} from "../../../../../util/format";
 
 export const LNDetails: FC<LNDetailProps> = (props) => {
+  const appCtx = useContext(AppContext);
   const { t } = useTranslation();
-  const details = props.details;
+  const { details } = props;
+  const [copyId] = useClipboard(details.id);
 
   const containerClasses =
     "m-2 py-1 flex overflow-hidden border-gray-400 border-b-2 text-left";
   const keyClasses = "w-1/2 text-gray-500 dark:text-gray-200";
-  const valueClasses =
-    "w-1/2 overflow-hidden overflow-ellipsis overflow-x-auto";
+  const valueClasses = "w-1/2 overflow-hidden overflow-x-auto mx-2";
 
-  const date = new Date(details.date * 1000).toLocaleString(); // epoch time => * 1000
+  const date = new Date(details.time_stamp * 1000).toLocaleString(); // epoch time => * 1000
 
-  const copyClipboardHandler = () => {
-    navigator.clipboard.writeText(details.hash);
-  };
+  const amount =
+    appCtx.unit === "BTC"
+      ? convertToString(appCtx.unit, convertMSatToBtc(details.amount))
+      : convertToString(appCtx.unit, convertMSatToSat(details.amount));
 
   return (
-    <div className="flex flex-col p-3 my-4">
+    <div className="flex flex-col py-3 my-4">
       <div className={containerClasses}>
         <div className={keyClasses}>{t("tx.txid")}</div>
-        <div className={valueClasses}>{details.hash}</div>
+        <div className={valueClasses}>{details.id}</div>
```

Comment:
Nit-pick as well: that's a lot of `div`s :)  
Not too important though. Could be changed later to something more semantic.  
Even switching the inner `div`s to `p`s might already work.

Reply:
Yeah, that's why I created #93 

The Onchain & LnDetails [see here](https://github.com/cstenglein/raspiblitz-web/tree/master/src/components/Home/TransactionCard/TransactionDetailModal) could also be merged into one now or even removed and only displayed in the detailmodal component.

Reply:
Don't wanna put in too much work into it if they will be deleted, but will do some semantic HTML there ;)